src/main/java/io/stargate/sgv2/jsonapi/api/model/command/builders/TableSortClauseBuilder.java

```diff
@@ -14,4 +16,22 @@ public TableSortClauseBuilder(TableSchemaObject table) {
   public SortClause buildAndValidate(ObjectNode sortNode) {
     return defaultBuildAndValidate(sortNode);
   }
+
+  @Override
+  protected String validateSortClausePath(String path) {
+    // Tables have few rules: but cannot be empty
+    if (path.isEmpty()) {
+      throw ErrorCodeV1.INVALID_SORT_CLAUSE_PATH.toApiException(
+          "path must be represented as a non-empty string");
+    }
+    // Do we use escaping for Tables? For sake of consistency I think so
```

Comment:
This is how code was working, just adding comment to raise the point we don't have to, really (since "."s have no special meaning for Column names.

Reply:
sure, maybe reword the comment to be more definitive